data/scripting/infrastructure.lua

```diff
@@ -189,3 +189,61 @@ function is_building(immovable)
       immovable.descr.type_name == "militarysite" or
       immovable.descr.type_name == "trainingsite"
 end
+
+-- RST
+-- .. function:: add_wares(player, warehouse, waretable)
+--
+--    Adds(subtracts) wares to the first warehouse of specified type
+--
+--    :arg player: the player to add wares to
+--    :arg warehouse: The type of warehouse (string) to add wares to
+--    :arg waretable: a table of pairs {ware = value}
+
+function add_wares(player, warehouse, waretable)
```

Comment:
I think we should rename it to `add_wares_to_warehouse` if it's sitting this far up. We might get naming conflicts later.

Reply:
Thanks for the rename. Good to go now :)